tests/acceptance/bootstrap/SearchContext.php

```diff
@@ -220,4 +218,75 @@ public function theSearchResultShouldContainEntriesWithHighlight(
 			);
 		}
 	}
+
+	/**
+	 * @When /^user "([^"]*)" searches for "([^"]*)" inside (folder|space) "([^"]*)"(?: in space "([^"]*)")? using the WebDAV API$/
```

Comment:
you can have multiple lines here. that should make this less scary
```suggestion
	 * @When /^user "([^"]*)" searches for "([^"]*)" inside (folder|space) "([^"]*)" using the WebDAV API$/
	 * @When /^user "([^"]*)" searches for "([^"]*)" inside (folder) "([^"]*)" in space "([^"]*)") using the WebDAV API$/
```